plugins/sql-language/src/main/java/org/polypheny/db/sql/language/ddl/SqlCreateView.java

```diff
@@ -197,25 +186,4 @@ public void unparse( SqlWriter writer, int leftPrec, int rightPrec ) {
     }
 
 
-    /*
-    public void execute( Context context ) {
-        final Pair<PolyphenyDbSchema, String> pair = SqlDdlNodes.schema( context, true, name );
-        final SchemaPlus schemaPlus = pair.left.plus();
-        for ( Function function : schemaPlus.getFunctions( pair.right ) ) {
-            if ( function.getParameters().isEmpty() ) {
-                if ( !getReplace() ) {
-                    throw SqlUtil.newContextException( name.getParserPosition(), RESOURCE.viewExists( pair.right ) );
-                }
-                pair.left.removeFunction( pair.right );
-            }
-        }
-        final SqlNode q = SqlDdlNodes.renameColumns( columnList, query );
-        final String sql = q.toSqlString( PolyphenyDbSqlDialect.DEFAULT ).getSql();
-        final ViewTableMacro viewTableMacro = ViewTable.viewMacro( schemaPlus, sql, pair.left.path( null ), context.getObjectPath(), false );
-        final TranslatableTable x = viewTableMacro.apply( ImmutableList.of() );
-        Util.discard( x );
-        schemaPlus.add( pair.right, viewTableMacro );
-    } */
```

Comment:
This needs to be enabled, otherwise, create view statements are NoOps. Why is this not detected by our tests?